client/app/layouts/splash.html

```diff
@@ -21,7 +21,7 @@
   <div layout="column" layout-align="center center" flex="noshrink" layout-margin>
     <div layout layout-align="center center">
       <logo></logo>
-      <h1 class="title">Foodsaving Worldwide</h1>
+      <heading align="center">Foodsaving<br>Worldwide</heading>
```

Comment:
heading isn't a proper element either. you can see them at https://developer.mozilla.org/en/docs/Web/HTML/Element. you can pick random words and they will behave like div elements but there should be good reason (like angular directives/components)